### PR TITLE
Sean Naleid: Sprint-Challenge-React-Wars

### PR DESCRIPTION
**Pull request by seanaleid**

### CODE REVIEW

File: starwars/src/components/StarWarsList.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+export default function StarWarsList(){
+    
+    //useState, initial value of an empty array and then setUseState to the response.data.results to loop over
+    const [people, setPeople] = useState([]);
+    console.log(`This is the empty array`, people);
+
+
+    useEffect(() =>{
+        axios.get(`https://swapi.co/api/people/`)
```
Comment (MrT3313) [inline starwars/src/components/StarWarsList.js:15]: try to do this in the APP and then pass into your `<StarWarsList>` props

File: starwars/src/components/StarWarsCard.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const Columns = styled.div`
+display: flex;
+flex-direction: column;
+justify-content: center;
+padding: 20px;
+
+`;
+
+const Font = styled.div`
```
Comment (MrT3313) [inline starwars/src/components/StarWarsCard.js:23]: cool implementation of styled--components

**Review by MrT3313 — COMMENTED** [review]
STYLINGGG ON SUPPPPPPERRRRR FLEEEEEEEK

- i like how you used your `<starWarsList>` as a bridge between your app and your individual cards BUT I would like to see you move the API call into the APP instead of having that in the bridge as well
- proper use of passing props
- Great use of styled components!

File: starwars/src/components/StarWarsList.js (hunk 1 above)
Comment (seanaleid) [reply starwars/src/components/StarWarsList.js:15]: Ok. I could also put it in another file and just pass it around as necessary, right?

**Pull request merged**